control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -82,6 +82,18 @@ var (
 	deleteCVORemovedResourcesOnce   sync.Once
 )
 
+const azureCCMScript = `
+#!/bin/bash
+set -o allexport
+if [[ -f /etc/kubernetes/apiserver-url.env ]]; then
+source /etc/kubernetes/apiserver-url.env
+fi
+exec /bin/azure-cloud-node-manager
```

Comment:
it doesn't look like the arguments below are passed to the call ... there's no line continuation chars `\`